wiki/es/es-Command-Line-Options.md

```diff
@@ -15,7 +15,7 @@ Por ejemplo, en Windows, para que un cliente utilice un archivo específico de c
 
 Para macOS, arranca una ventana de Terminal y ejecuta Jamulus con las opciones deseadas como sigue:
 
-/Applications/Jamulus.app/Contents/MacOS/Jamulus --inifile "/ruta/a/myinifile.ini"
+```shell /Applications/Jamulus.app/Contents/MacOS/Jamulus --inifile "/path/to/myinifile.ini" ```
```

Comment:
Long commands could cause an overflow on mobile devices. We might need to use:

```suggestion
```shell
/Applications/Jamulus.app/Contents/MacOS/Jamulus --inifile "/path/to/myinifile.ini"
```
```
which generates a scroll-able code box